src/label_studio_sdk/converter/converter.py

```diff
@@ -216,14 +242,16 @@ def convert(self, input_data, output_data, format, is_dir=True, **kwargs):
             )
         elif format == Format.CONLL2003:
             self.convert_to_conll2003(input_data, output_data, is_dir=is_dir)
-        elif format == Format.COCO:
+        elif format in [Format.COCO, Format.COCO_WITH_IMAGES]:
             image_dir = kwargs.get("image_dir")
+            self.download_resources = format == Format.COCO_WITH_IMAGES
             self.convert_to_coco(
                 input_data, output_data, output_image_dir=image_dir, is_dir=is_dir
             )
-        elif format == Format.YOLO or format == Format.YOLO_OBB:
+        elif format in [Format.YOLO, Format.YOLO_OBB, Format.YOLO_OBB_WITH_IMAGES, Format.YOLO_WITH_IMAGES]:
             image_dir = kwargs.get("image_dir")
             label_dir = kwargs.get("label_dir")
+            self.download_resources = format in [Format.YOLO_WITH_IMAGES, Format.YOLO_OBB_WITH_IMAGES]
             self.convert_to_yolo(
```

Comment:
further down in this call, `is_obb` should be `True` if the format is `YOLO_OBB_WITH_IMAGES`